subsys/power/policy/Kconfig

```diff
@@ -0,0 +1,69 @@
+
+choice
+	prompt "PM Policy"
```

Comment:
I assume that this is in fact "Idle State management".
Could we name things accordingly in order to be prepared for other features, like Frequency/Voltage scaling etc.?

For example:
- Power Management Options:
  - Idle State Management
     - Idle state handling policy
  - CPU Frequency scaling [example]
  - ....